src/app/pages/ConnectorDetailsPage/components/ConfigurationTab/ConfigurationTab.tsx

```diff
@@ -67,7 +67,8 @@ const diff = (
   return Object.keys(newConfig).filter((key) => {
     if (oldConfig === undefined || oldConfig[key] === newConfig[key])
       return false;
-    if (oldConfig[key] === {} || newConfig[key] === '') return false;
+    if (JSON.stringify(oldConfig[key]) === '{}' || newConfig[key] === '')
```

Comment:
I changed this line because I was seeing either a compilation or runtime warning about `oldConfig[key] === {}` not being a safe comparison, I think this still matches the intent.